packages/address-book/address-book/arbitrum/tokens/tokens.ts

```diff
@@ -31,14 +31,25 @@ const _tokens = {
   },
   USDC: {
     name: 'USD Coin',
-    address: '0xFF970A61A04b1cA14834A43f5dE4533eBDDB5CC8',
+    address: '0xaf88d065e77c8cC2239327C5EDb3A432268e5831',
     symbol: 'USDC',
     decimals: 6,
-    website: 'https://www.circle.com/usdc',
+    website: 'https://www.centre.io/',
```

Comment:
isn't this native USDC? shouldn't it be circle's site?

Reply:
arbiscan show centre.io as token website 
<img width="1391" alt="image" src="https://github.com/beefyfinance/beefy-api/assets/88328748/eb55ecc9-aa01-4303-b22d-1545d9ae4fc9">